packages/lesswrong/lib/routes.ts

```diff
@@ -775,16 +775,10 @@ const forumSpecificRoutes = forumSelect<Route[]>({
       redirect: () => '/saved'
     },
     {
-      name: 'savedPosts',
+      name: 'savedAndRead',
       path: '/saved',
       componentName: 'BookmarksPage',
-      title: 'Saved Posts',
-    },
-    {
-      name: 'readHistory',
-      path: '/history',
```

Comment:
Maybe we should add a redirect from /history to /saved? (Might not be important though - I never realised /history existed, but maybe somebody uses it)

Reply:
Yeah it was pretty hidden so I don't think it's worth adding